content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors (ACEis) and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.
```

Comment:
This is an assertion that could have a ref. If you took out the specific claims about number of prescriptions I think it could go unsourced.

Reply:
I am unsure whether I can edit my pull request directly, so I am adding the reference here: @url:https://clincalc.com/DrugStats/Drugs/Lisinopril

Reply:
Here is a second reference, this one addressing ARBs: @url:https://clincalc.com/DrugStats/Drugs/LosartanPotassium. One could also cite this publication that we wrote last year: https://www.nejm.org/doi/full/10.1056/nejmp1901657.

Reply:
These medications are commonly used for the treatment of hypertension, and somewhat less commonly, but still widely for the treatment of heart failure, kidney disease.

Reply:
The authors did try to take certain co-existing conditions into account, but in registry data, that is a messy proposition.

Reply:
I need to read the paper more thoroughly, but I do not think that it is correct to say that they found odds of death of .33.  That would mean a probability of death of 25%.  What they said was that there was an odds RATIO of 0.33.  That is very different.